src/app/games/new-game/NewGameRules.ts

```diff
@@ -94,6 +94,10 @@ export class NewGameRules extends Rules<NewGameMove, NewGameState, NewGameLegali
      * @returns a GameStatus (ZERO_WON, ONE_WON, DRAW, ONGOING)
      */
     public getGameStatus(node: NewGameNode): GameStatus {
-        return GameStatus.DRAW;
+        if (node.gameState.turn < 42) {
+            return GameStatus.ONGOING;
+        } else {
+            return GameStatus.DRAW;
+        }
```

Comment:
why ? (why not j'imagine, ou bien une raison particulière ?)

Reply:
c'est mieux comme ça non ?
Plutôt que "draw dès le tour 1"

Reply:
ouais 'fin c'est un dummy jeux, faut qu'il soit au plus simple, on veut pas d'un second MinimaxTesting !

Reply:
bah ça reste super simple mais qu'ils aient un exemple quand y a plusieurs contexte est pas mauvais quoi !